igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/TravisBuildMonitor.groovy

```diff
@@ -230,4 +195,50 @@ class TravisBuildMonitor implements PollingMonitor{
         results
 
     }
+
+    private void sendEventForBuild(Repo repo, String master, TravisService travisService) {
+        if (echoService) {
+            log.info "pushing event for ${master}:${repo.slug}:${repo.lastBuildNumber}"
+
+            GenericProject project = new GenericProject(repo.slug, TravisBuildConverter.genericBuild(repo, travisService.baseUrl))
+            echoService.postEvent(
+                new GenericBuildEvent(content: new GenericBuildContent(project: project, master: master, type: 'travis'))
+            )
+
+        }
+        Commit commit = travisService.getCommit(repo.slug, repo.lastBuildNumber)
+        if (commit) {
+            String branchedSlug = "${repo.slug}/${commit.branchNameWithTagHandling()}"
+            buildCache.setLastBuild(master, branchedSlug, repo.lastBuildNumber, repo.lastBuildState == BUILD_IN_PROGRESS)
+            if (echoService) {
+                log.info "pushing event for ${master}:${branchedSlug}:${repo.lastBuildNumber}"
+
+                GenericProject project = new GenericProject(branchedSlug, TravisBuildConverter.genericBuild(repo, travisService.baseUrl))
+                echoService.postEvent(
+                    new GenericBuildEvent(content: new GenericBuildContent(project: project, master: master, type: 'travis'))
+                )
+
+            }
+        }
+    }
+
+    private void pushEventsForMissingBuilds(Repo repo, Map cachedBuild, String master, TravisService travisService) {
+        int currentBuild = repo.lastBuildNumber
+        int lastBuild = Integer.valueOf(cachedBuild.lastBuildLabel)
+
+        if (lastBuild+1 < currentBuild) {
```

Comment:
This routine might read easier if you were to capture the 'build event threshold" as a constant and then create a local variable for the `lastBuild + newBuildEventThreshold` (that might not be a great name, but you get the idea...). It would let you avoid doing the arithmetic twice and would identify the `1` as some meaningful (configurable) value.